src/SfxWeb/src/app/Common/Constants.ts

```diff
@@ -75,7 +75,9 @@ export class Constants {
     public static SvgTransitionDurationSlow = 600;
 
     public static executingInfraJobsSuggestion = 'If the repair task corresponding to Infrastructure updates is stuck in Preparing for long, check the Repair Task page.';
-    public static pendingInfraJobsSuggestion = 'Pending jobs when other jobs are executing are not executed because of throttling policy in Infrastructure Service.';
+    public static pendingInfraJobsSuggestion = 'Pending updates when 2 or more updates are in Active state are not executed because of throttling policy in Infrastructure Service.';
+    public static longExecutingInfraJobsSuggestion = 'Infrastructure Jobs are in Executing state for more than 1 hour. Engage Compute platform teams if its taking too much time.';
```

Comment:
"Infrastructure Jobs are in Executing state for more than 1 hour after SF approval."